compiler/src/dotty/tools/dotc/parsing/Parsers.scala

```diff
@@ -4,23 +4,24 @@ package parsing
 
 import scala.collection.mutable.ListBuffer
 import scala.collection.immutable.BitSet
-import util.{ SourceFile, SourcePosition }
+import util.{SourceFile, SourcePosition}
```

Comment:
also - these changes here - are they done by your editor or by you? I'd leave them out of the diff